.github/workflows/pr-closed.yml

```
@@ -0,0 +1,32 @@
+name: 🎉 PR closed
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  thank-you:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post Thank You Comment
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        env:
+            comment: |
+              ## Merge Successful
+
+              Your pull request has been successfully merged. We greatly appreciate your time and effort in making this project better! 💪
+
+              Happy coding! ✨
```

I'd prefer to keep things simple:

```suggestion
            comment: Thanks for helping make WXT better!
```